.github/workflows/security-scan.yml

```
@@ -119,7 +119,15 @@ jobs:
 
     - name: Run TypeScript security-only check
       working-directory: Packages/src/TypeScriptServer~
-      run: npm run security:sarif-only || echo "Security ESLint completed with issues - SARIF file generated"
+      run: |
+        npm run security:sarif-only || echo "Security ESLint completed with issues - SARIF file generated"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)